core/src/main/scala/cats/syntax/all.scala

```diff
@@ -93,3 +94,5 @@ trait AllSyntaxBinCompat4
 trait AllSyntaxBinCompat5 extends ParallelBitraverseSyntax
 
 trait AllSyntaxBinCompat6 extends ParallelUnorderedTraverseSyntax
+
+trait AllSyntaxBinCompat7 extends FunctorSyntaxBinCompat0
```

Comment:
I can't remember why some syntax traits are package private and some aren't.  I can't think of a reason this couldn't and shouldn't be, but I also was not very active in developing this idiom.

Reply:
@rossabaker As far as I know there's not a good answer to this question—like a lot of stuff in Cats it's just the result of a lot of uncoordinated decisions. I opened a related issue [here](https://github.com/typelevel/cats/issues/3008) a few months ago.

Reply:
@gagandeepkalra has been through a lot of incidental distractions already. I'm good with punting on it for the purposes of this ticket.

Reply:
thank you